src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,152 @@
+$theme-focusHighlight: #1c64f2;
+$theme-primaryBackground: #1c325e;
+$theme-whiteColor: #ffffff;
+$theme-bannerFontColor: #f1416c;
+$theme-secondaryBackground: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-headingColor: #6b7280;
+$theme-inputFieldBorder: #d1d5db;
+$theme-nonActiveButtonBackground: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.88rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-primaryBackground;
+    width: 84%;
+    height: 12%;
+    color: $theme-whiteColor;
+    padding: 2rem;
+    border-radius: $radius;
+}
+
+.standupBanner .bannerPara {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.24rem;
+    font-weight: 600;
+}
+
+.totalMissedUpdate {
+    color: $theme-bannerFontColor;
+    text-decoration: underline;
+    margin: 0 0.4rem;
+}
+
+.standupUpdateContainer {
+    width: 50.8%;
+    margin-top: 4rem;
+}
+
+.standupForm {
+    margin-top: 2.2rem;
+}
+
+.standupTitle {
+    color: $theme-secondaryBackground;
+    font-family: 'Inter', sans-serif;
+    font-size: 2.2rem;
+    padding-left: 0.8rem;
+}
+
+.formFields {
+    margin-bottom: 1.25rem;
+    border: none;
+}
+
+.updateHeading {
+    color: $theme-headingColor;
+    font-size: 1.4rem;
+    font-family: 'Inter', sans-serif;
+}
+
+.inputField {
+    height: 4rem;
+    width: 100%;
+    border-radius: $radius;
+    border: 1px solid $theme-inputFieldBorder;
+    background: $theme-whiteColor;
+    padding: 1rem;
+    font-weight: 400;
+    font-size: 0.88rem;
+    line-height: 150%;
+    font-family: 'Inter', sans-serif;
+    margin-top: 0.8rem;
+    margin-bottom: 2.4rem;
+}
+
+.inputField:focus {
+    border: 1px solid $theme-focusHighlight;
+    outline: none;
+}
+
+.submitButton {
+    width: 14rem;
+    height: 4rem;
+    border-radius: $radius;
+    font-weight: 400;
+    font-size: 0.88rem;
+    line-height: 150%;
+    font-family: 'Inter', sans-serif;
+    border: none;
+    cursor: pointer;
+    letter-spacing: 0.8px;
+    margin-left: 1rem;
+}
+
+.submitButton:active {
+    background: $theme-secondaryBackground;
+    color: $theme-whiteColor;
+}
+
+.activeButton {
+    background: $theme-secondaryBackground;
+    color: $theme-whiteColor;
+}
+
+.nonActiveButton {
```

Comment:
can we use classes like button enabled and button disabled the current ones don't sound good (Just a personal preference, if others are ok with the current, then we can stick to it.)